hyundai_kia_connect_api/HyundaiBlueLinkAPIUSA.py

```diff
@@ -386,6 +386,9 @@ def _update_vehicle_properties(self, vehicle: Vehicle, state: dict) -> None:
         vehicle.ev_battery_is_plugged_in = get_child_value(
             state, "vehicleStatus.evStatus.batteryPlugin"
         )
+        vehicle.ev_charging_current = get_child_value(
+            state, "vehicleStatus.evStatus.batteryStndChrgPower"
```

Comment:
shouldn't this be vehicleStatus.evStatus.batteryPower.batteryStndChrgPower?
according to https://github.com/Hyundai-Kia-Connect/kia_uvo/issues/961

Reply:
Good catch. Worth confirming what region that screenshot is from.